tests/zquantum/qaoa/ansatzes/x_ansatz_test.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+import sympy
+from openfermion import IsingOperator, QubitOperator
+from zquantum.core.circuits import CNOT, RZ, Circuit, H
+from zquantum.core.interfaces.ansatz_test import AnsatzTests
+from zquantum.core.openfermion import change_operator_type
+from zquantum.core.utils import compare_unitary
+from zquantum.qaoa.ansatzes.x_ansatz import QAOAXAnsatz, create_x_qaoa_circuits
+
+
+def create_thetas(number_of_params):
+    return sympy.symbols(f"theta_:{number_of_params}")
+
+
+def create_symbols_map(number_of_params):
+    symbols_map = {}
+    thetas = create_thetas(number_of_params)
+    for i in range(len(thetas)):
+        symbols_map[thetas[i]] = 0.5
+    return symbols_map
+
+
+def create_target_unitary(number_of_params, k_body_depth=1):
+    thetas = create_thetas(number_of_params)
+    symbols_map = create_symbols_map(number_of_params)
+
+    target_circuit = Circuit()
+    target_circuit += H(0)
```

Comment:
I think you could do thas in a more concise way, e.g. by creating multiple circuits with list of gates and then adding them together:
```target_circuit = Circuit([H(0), RZ(2*thetas[0])(0), ...]
second_layer = Circuit(...) ```